client/src/components/create-assignment/ParentAssignment.js

```diff
@@ -27,6 +30,17 @@ const ParentAssignment = ({ indices, addSubAssignments, setAssignments, assignme
     setOpen(!open);
   };
 
+  /* Functions to get the formula attributes.
+     formulaId specifies the formula that is used to calculate this assignmnet's garde,
+     subFormulaAttributeNames are the attributes that need to be specified for the direct sub assignments of this assignments,
+     so that the grade for this assignment can be calculated. 
+     Observe that formulaAttributeNames that is as a parameter for this component are the attributes that need to specified for this assignmnet,
+     so that the grade of this assignment's parent assignment can be calculated.
+  */
+  const formulaId = assignmentServices.getProperty(indices, assignments, 'formulaId');
+  const formulaName = formulasService.getFormula(formulaId);
```

Comment:
Should this getFormula return the whole formula as an object and not just the name?

Reply:
I could actually change the function name to `getFormulaName`?